ocd_backend/transformers/organizations.py

```diff
@@ -18,79 +22,79 @@ def transform_contact_details(data):
-            'source': self.source_definition['key'],
-            'supplier': 'allmanak',
-            'collection': 'governmental_organization',
-        }
-
-        object_model = TopLevelOrganization(self.original_item['systemid'],
-                                            self.source_definition,
-                                            **source_defaults)
-        object_model.entity = self.entity
-        object_model.classification = u'Province'
-        object_model.collection = self.source_definition['key']
-        object_model.name = unicode(self.original_item['naam'])
-        object_model.description = self.original_item['omvatplaats']
-        # object_model.contact_details = transform_contact_details(self.original_item['contact'])
-
-        return object_model
-
-
-class PartyItem(BaseTransformer):
-    """
-    Creates a Party (Organization) item.
-    """
-
-    def transform(self):
-        source_defaults = {
-            'source': self.source_definition['key'],
-            'supplier': 'allmanak',
-            'collection': 'party',
-        }
-
-        # When the Allmanak implements parties as entities, the entity ID should be used
-        object_model = Organization(self.original_item['partij'],
-                                    self.source_definition,
-                                    **source_defaults)
-        object_model.entity = self.entity
-        object_model.has_organization_name = TopLevelOrganization(self.source_definition['key'], **source_defaults)
-        object_model.has_organization_name.merge(collection=self.source_definition['key'])
-        object_model.collection = self.source_definition['key'] + '-' + self.original_item['partij']
-        object_model.name = self.original_item['partij']
-        object_model.classification = 'Party'
-        object_model.subOrganizationOf = TopLevelOrganization(self.source_definition['key'], **source_defaults)
-        object_model.subOrganizationOf.merge(collection=self.source_definition['key'])
-
-        return object_model
+@celery_app.task(bind=True, base=BaseTransformer, autoretry_for=(Exception,), retry_backoff=True)
+def municipality_organization_item(self, content_type, raw_item, entity, source_item, **kwargs):
+    original_item = self.deserialize_item(content_type, raw_item)
+    source_definition = kwargs['source_definition']
+
+    source_defaults = {
+        'source': source_definition['key'],
+        'supplier': 'allmanak',
+        'collection': 'governmental_organization',
+    }
+
+    object_model = TopLevelOrganization(original_item['systemid'],
+                                        source_definition,
+                                        **source_defaults)
+    object_model.entity = entity
+    object_model.classification = u'Municipality'
+    object_model.collection = source_definition['key']
+    object_model.name = ' '.join([source_definition.get('municipality_prefix', ''), unicode(original_item['naam'])])
```

Comment:
To create full municipality names in the case of "Amsterdam + Stadsdeel", where "Amsterdam" is the prefix.

Reply:
We already have `source_definition.get('sitename')` which is exactly the same value. Wouldn't that suffice?

Reply:
The names in Notubiz are listed as "Stadsdeel X", so if we want to stay as close to the source data as possible a prefix seems like the best solution. If we used `sitename` it would become "Amsterdam_noord Stadsdeel Noord".